search/quality_eval.go

```diff
@@ -245,10 +258,15 @@ type HitSource struct {
 // https://archive.kbb1.com/he/events/meals
 // https://archive.kbb1.com/he/events/friends-gatherings
 // https://archive.kbb1.com/he/events?year=2013
-func ParseExpectation(e string) Expectation {
+func ParseExpectation(e string, db *sql.DB) Expectation {
+	originalE := e
 	if e == "" {
 		return Expectation{ET_EMPTY, "", nil, e}
 	}
+	takeLatest := strings.HasPrefix(e, PREFIX_LATEST)
```

Comment:
Lower "e" so that [Latest] will also work well.

Reply:
done